code/modules/mob/living/carbon/human/say.dm

```diff
@@ -30,6 +30,8 @@
 	// how do species that don't breathe talk? magic, that's what.
 	if(!HAS_TRAIT_FROM(src, TRAIT_NOBREATH, SPECIES_TRAIT) && !getorganslot(ORGAN_SLOT_LUNGS))
 		return FALSE
+	if(!dna?.species.speak_no_tongue && !getorganslot(ORGAN_SLOT_TONGUE))  // candy-etherware
```

Comment:
This is not logically equivalent. Change to
`if(dna?.species && !dna?.species.speak_no_tongue)`

Otherwise if `dna` or `species` is null you won't be able to talk without a tongue.